code/modules/mob/mob_defines.dm

```diff
@@ -213,3 +213,7 @@
 
 	/// A mock client, provided by tests and friends
 	var/datum/client_interface/mock_client
+
+	/// two vars for determining emote cooldown
+	var/emote_used = 0
+	var/emote_cooldown = 0.5 SECONDS
```

Comment:
```suggestion
	var/static/emote_cooldown = 0.5 SECONDS
```

Reply:
Honestly I feel like it should just be a define.

Reply:
Depending on how the system is written, each individual emote has a cooldown or there is just a global emote cooldown.

Reply:
Yeah the tg system puts an individual cooldown for each emote, which does still allow you to spam quite a lot, this method will simply put a cooldown on every emote.